### DISCUSSION TITLE
Why are the metrics values ​​with observed different from those without observed?

### DISCUSSION THREAD

**Discussion by hochan222** [GoogleChrome/lighthouse#14190] [Q&A] [answered]
Hello, I left an issue because I couldn't find a cool answer while using the lighthouse api.

**Why are the metrics values ​​with observed different from those without observed?** (What does observed mean?)

The following can be obtained from metrics from audits that can be obtained with the lighthouse API. You can see the difference between firstContentfulPaint at 982 and observedFirstContentfulPaint at 741. Why is this difference happening? (Actually, I want to get the value of load, but the only load value I can get from the api is observedLoad, so I came to this question. Lighthouse is also generating reports using non-observed figures. If so, how can I know the load..?)

![image](https://user-images.githubusercontent.com/22424891/177108466-5686e935-6581-482f-9fae-0baa4c9df707.png)

Another question.. observedLoad is sometimes measured as 3096 and observedLargestContentfulPaint is sometimes measured as 4464. observedLargestContentfulPaint is measured to be larger than observedLoad. I wonder if the load metric only means loading the resource, not the completion of rendering(paint) in the browser.

Thank you :)

**Comment by adamraine** [comment, marked as answer]
> Why are the metrics values ​​with observed different from those without observed? (What does observed mean?)

By default, Lighthouse will load the page normally without any throttling applied, and then compute what the metric values would have been if there was throttling. The variables named `observed*` represent the metric values *before* throttling is simulated.

See https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md for more info

> Another question.. observedLoad is sometimes measured as 3096 and observedLargestContentfulPaint is sometimes measured as 4464. observedLargestContentfulPaint is measured to be larger than observedLoad. I wonder if the load metric only means loading the resource, not the completion of rendering(paint) in the browser.

`observedLoad` refers to the timing of the [page onload event](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event). This is a different metric from LCP and we don't compute a throttled version for it.

**Reply by hochan222** [reply]
Thanks for the detailed reply. It was very helpful for my understanding.

Have a nice day 🙇🙇
